build-tools/packages/build-cli/docs/typetestDetails.md

```diff
@@ -57,7 +57,7 @@ but this can be suppressed with the `--no-generateInName` flag.
 Type tests can be configured to use different baseline versions on a given branch depending on the type of release that
 the branch is designated for. For example, for the client release group, the _next_ branch is the _major version series
 branch_ and _main_ is the _minor version series branch_. This can be declared in the root package.json for the release
-group, in the `fluidBuild.repoPackages` section. For example, the following configuration designates the _main_ and
+group, in the `fluidBuild.branchReleaseTypes` section. For example, the following configuration designates the _main_ and
```

Comment:
Good catch, thanks! This illustrates the dangers of writing docs before code review and forgetting to update the docs with the code review updates... 😄